optimize/self-managed/optimize-deployment/migration-update/3.13_8.5-to-3.14_8.6.md

```diff
@@ -17,20 +17,24 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
-## Changes in supported environments
+## Changes in behaviour
 
 ### Telemetry
 
-Optimize no longer gathers telemetry data. This change includes removing it from UI and ElasticSearch. The associated key in configuration file (`telemetry.telemetryEndpoint`) was removed.
+Optimize no longer gathers telemetry data. This change includes removing it from UI and ElasticSearch. The associated key in the configuration file (`telemetry.telemetryEndpoint`) was removed.
+
+## Changes in supported environments
 
 ### Camunda 7
 
-In Optimize, the minimum supported version of `Java` is 21.
+Optimize now requires at least Camunda 7 `7.20.0` and supports up to `7.22.0+`. Camunda 7 `7.19.x` is not supported anymore.
```

Comment:
@conceptualshark How do you feel about the 7 proceeding `7.20.0` here? I have mixed feelings.

Reply:
It's a weird one that feels more marketing-oriented; do we always refer to the product distinction as "Camunda 7" vs "Camunda 8" and not simply say "Camunda"? In the case of the former, even if it's a little clunky, I think "Camunda 7 `7.20.0`" to double-down on the distinction makes sense.